deployment/helm/README.md

```diff
@@ -44,8 +44,32 @@ helm init
 
 ```bash
 cd <DITTO_PATH>/deployment/helm/
+kubectl create namespace dittons
 helm update dependencies
```

Comment:
Path is missing here: helm dependencies update ./eclipse-ditto/

Reply:
Fixed, good catch!